README.md

```diff
@@ -104,6 +104,39 @@ Terms are compared under the semantics of _toEqualRdfTerm_.
 expect([namedNode('t1'), namedNode('t2')]).toEqualRdfTermArray([namedNode('t2'), namedNode('t3')]);
 ```
 
+#### toBeRdfDatasetContaining
+
+Check if a dataset contains one or more quads.
```

Comment:
Could you mention here that any number of quads can be passed, and that *all* of these quads must be present in the dataset before the test passes? (as opposed to just one of the given quads)